openmetadata-ui/src/main/resources/ui/src/components/DataAssetSummaryPanel/DataAssetSummaryPanel.tsx

```diff
@@ -175,16 +175,20 @@ export const DataAssetSummaryPanel = ({
       case EntityType.TOPIC:
         return (
           <>
-            <Row
-              className="p-md border-radius-card summary-panel-card"
-              gutter={[0, 4]}>
-              <Col span={24}>
-                <CommonEntitySummaryInfo
-                  componentType={componentType}
-                  entityInfo={entityInfo}
-                />
-              </Col>
-            </Row>
+            {entityInfo.some((info) =>
+              info.visible?.includes(componentType)
+            ) && (
```

Comment:
It may possible for other entities?

Reply:
For other summary panels we have a placeholder of no data like this
<img width="417" alt="Screenshot 2025-03-18 at 11 34 44 AM" src="https://github.com/user-attachments/assets/21d9698b-f7e3-4fed-b3f5-3ddf85bcfe93" />